index.html

```diff
@@ -36,98 +36,95 @@ <h2>Brigade is a tool for running scriptable, automated tasks in the cloud &mdas
         <div class="medium-7 columns">
 
           <h2>Simple, powerful pipes</h2>
-          <p>Each project gets a <code>brigade.js</code> config file, which is where you can write dynamic, interwoven pipelines and tasks for your Kubernetes cluster. Leave your YAML at home!</p>
+          <p>Each project uses a script file written in either JavaScript or TypeScript, which is where you can write dynamic, interwoven pipelines and tasks to handle inbound events. Leave your YAML at home!</p>
```

Comment:
imho, I would shorten this, de-emphasize pipelines (so we're not inviting unwelcome comparisons to Argo) and put the emphasis more squarely on the events. Something like this:

> Each project uses JavaScript or TypeScript to implement dynamic event-handling.

"Leave your YAML at home!" might, unfortunately, sound like a false promise since project definitions are still written in YAML. This being the case, it might be better to cut that as well.

Reply:
Updated.